packages/components/src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -67,22 +25,27 @@ const ProgressBar = React.forwardRef<HTMLDivElement, ProgressBarProps>(
     const borderRadius = size === "large" ? "borderRadius50" : "borderRadius10";
 
     return (
-      <StyledProgress
+      <Box.div
         as={Root}
-        background={backgroundColor}
-        h={size === "large" ? "space40" : "space25"}
-        radius={borderRadius}
+        backgroundColor={backgroundColor || "colorBackgroundWeak"}
```

Comment:
Can we move the default into the params similar to `size`? The stories don't seem to be set up properly (or just broken) for it to show right now but storybook will then show the default value in the props table for the component. Also, just makes it easier to know what the default values for props are.

```    { value, size = "small", indicatorColor = "colorBackgroundPrimaryWeak", backgroundColor = "colorBackgroundWeak", ...props },
```

Reply:
oh good call! Just changed!